source/user-manual/reference/daemons/ossec-authd.rst

```diff
@@ -5,7 +5,7 @@
 ossec-authd
 ===========
 
-The ``ossec-authd`` program can automatically add an agent to a Wazuh manager and provide the key to the agent. It's used along with the :ref:`agent-auth` application. The program creates an agent with an IP address of ``any`` instead of using a specific IP address.
+The ``ossec-authd`` program can automatically add an agent to a Wazuh manager and provide the key to the agent. It's used along with the agent-auth application. The program creates an agent with an IP address of ``any`` instead of using a specific IP address.
```

Comment:
```suggestion
The ``ossec-authd`` program can automatically add a Wazuh agent to a Wazuh manager and provide the key to the agent. It's used along with the ``agent-auth`` application. The program creates an agent with an IP address of ``any`` instead of using a specific IP address.
```